docs/howto/openstack/nova/new-server.md

```diff
@@ -23,90 +23,132 @@ to work with the OpenStack CLI, please do not forget to [source the RC
-    _Ubuntu 22.04 Jammy Jellyfish 20220810_ in particular.
+    Below the region selection is a drop-down menu where you must select a
+    _server profile_. Currently, four profiles are available:
 
-    ![Boot source](assets/new-server/shot-03.png)
+    *  _Generic_,
+    * _High CPU_,
+    * _Low Latency Disk_, and
+    *  _GPU_.
 
-    Next, make sure _Boot Target_ is set to _Volume (Recommended)_.
-    Regarding the server's CPU core count and amount of memory, set the
-    [_Flavor_](../../../reference/flavors/index.md) accordingly.
-    The default flavor specifies a server with 1 CPU core and 1GB of RAM.
-    You can start with that or select a different configuration by clicking
-    the dropdown menu at the right of _Flavor_. Please note that, depending on
-    the chosen flavor, the estimated monthly cost of the server changes. (While a
-    server is shut off you are still getting charged for it, but less so.)
-    At any time, this estimated cost is displayed in the green rectangular area
-    at the top. Something else that affects the cost is the size of the root
-    device. Take a look at the _Size_ parameter below, and notice the
-    default (in
-    [gibibytes](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary))).
-    You may leave the root device size unchanged, or modify it to be lower
-    or higher than the default.
-
-    When, at a later time, you decide to delete the server, you can
-    do so but **keep** its boot volume (you may want, for example, to
-    attach the exact same volume to a new server). Just leave the _Delete
-    on termination_ option disabled if you want this kind of flexibility.
-    On the other hand, if you want your root volume to be automatically
-    deleted when the server terminates, enable _Delete on termination_.
-    Use this option with caution.
-
-    Finally, you may enable _Disaster recovery_ for the server. If
-    you do, then daily server snapshots will be created, and you will have
-    the option for easy and fast rollups to previous snapshots. Please be
-    aware that enabling this option increases the server's monthly estimated
-    cost (again, it is displayed in the green rectangular area at the top).
+    Selecting one of those gives you access to a subset of all available
+    [_flavors_](../../../reference/flavors/index.md).
+
+    As the profile names suggest, each profile but the _Generic_ points to a
+    set of flavors suitable for servers intended to run specific types of
+    applications:
+
+    * The _Generic_ profile is for general-purpose cloud servers.
+    * The _CPU_ profile is for servers designed to host CPU-intensive
+      applications.
+    * The _Low Latency Disk_ profile is for servers that boot off of a local
+      low-latency disk.
+    * The _GPU_ profile is for servers that boot off of a local SSD disk
+      and, most importantly, have direct access to an NVIDIA vGPU.
+
+    We should point out that _not_ all profiles may be available in any
```

Comment:
Micro-nit: reading this out loud it seems odd that "not" is emphasised here. I think you have two options:

1. Emphasise "all" instead:
```suggestion
    We should point out that not _all_ profiles may be available in any
```
2. Drop the emphasis altogether:
```suggestion
    We should point out that not all profiles may be available in any
```

Reply:
It is odd indeed. And since some emphasis wouldn't hurt here, I am emphasizing "all".